tests/Unit.Tests/ViewFormatters/CourseExtensions.Tests.cs

```diff
@@ -60,5 +64,136 @@ public void FormattedDistanceTest(double? input, string expectedDisplayValue)
+        [Test]
+        [TestCase("Drama (with English)")]
+        [TestCase("Drama (with English)")]
+        [TestCase("Media Studies (with English)")]
+        [TestCase("PE with Maths")]
+        [TestCase("Physical Education (with EBacc Subject)")]
+        [TestCase("Physical Education (with English 11-16)")]
+        [TestCase("Physical Education (with Geography 11-16)")]
+        [TestCase("Physical Education (with Mathematics 11-16)")]
+        [TestCase("Physical Education with Biology")]
+        [TestCase("Physical Education with Chemistry")]
+        [TestCase("Physical Education with Computer Science")]
+        [TestCase("Physical Education with English")]
+        [TestCase("Physical Education with French")]
+        [TestCase("Physical Education with Geography")]
+        [TestCase("Physical Education with History")]
+        [TestCase("Physical Education with Mathematics")]
+        [TestCase("Physical Education with Maths specialism")]
+        [TestCase("Physical Education with Physics")]
+        [TestCase("Physical Education with Spanish")]
+        public void OverriddenFunding(string courseName)
+        {
+            using (new AssertionScope())
+            {
+                var course1 = CourseBuilder.BuildCourse(courseName)
+                    .AddBursarySubject()
+                    .AddBursaryAndScholarshipSubject();
+                course1.FundingOptions().Should().Be(CourseExtensions.FundingOption_StudentFinance);
+                var course2 = CourseBuilder.BuildCourse(courseName)
+                    .AddSubject()
+                    .AddBursaryAndScholarshipSubject();
+                course2.FundingOptions().Should().Be(CourseExtensions.FundingOption_StudentFinance);
+                var course3 = CourseBuilder.BuildCourse(courseName)
+                    .AddSubject()
+                    .AddBursarySubject();
+                course3.FundingOptions().Should().Be(CourseExtensions.FundingOption_StudentFinance);
+                var course4 = CourseBuilder.BuildCourse(courseName)
+                    .AddSubject();
+                course4.FundingOptions().Should().Be(CourseExtensions.FundingOption_StudentFinance);
+                var course5 = CourseBuilder.BuildCourse(courseName)
+                    .AddSubject()
+                    .AddSubject();
+                course5.FundingOptions().Should().Be(CourseExtensions.FundingOption_StudentFinance);
+
+                var course6 = CourseBuilder.BuildCourse(courseName)
+                    .AddBursarySubject();
+                course6.FundingOptions().Should().Be(CourseExtensions.FundingOption_Bursary);
+                var course7 = CourseBuilder.BuildCourse(courseName)
+                    .AddBursaryAndScholarshipSubject();
+                course7.FundingOptions().Should().Be(CourseExtensions.FundingOption_Scholarship);
+            }
```

Comment:
var course8 = CourseBuilder.BuildCourse(courseName)
                    .AddSalary();
                course8.FundingOptions().Should().Be(CourseExtensions.FundingOption_Salary);

just for completeness